src/applications/personalization/dashboard/tests/containers/YourApplications.unit.spec.jsx

```diff
@@ -24,6 +24,15 @@ const defaultProps = {
 };
 
 const sipEnabledForm = {
+  form: VA_FORM_IDS.FORM_10_10EZ,
+  metadata: {
+    lastUpdated: '2019-04-24T00:00:00.000-06:00',
+    // create an expiration date one week in the future
+    expiresAt: new Date(Date.now() + 7 * 24 * 60 * 60 * 1000).toISOString(),
```

Comment:
previously this timestamp was hardcoded to a date in the past, but that would cause a test failure now